.pipelines/cni/pipeline.yaml

```diff
@@ -140,6 +156,34 @@ stages:
       vmSize: Standard_DS4_v2
       arch: amd64
 
+  - template: singletenancy/linux-cniv2-template.yaml
+    parameters:
+      name: mariner_linux_overlay
+      clusterType: overlay-byocni-up
+      clusterName: mariner-cniv2-overlay
+      nodeCount: ${LINUX_NODE_COUNT}
+      vmSize: Standard_DS4_v2

```

Comment:
I think we made the `vmSize` as a parameter for cni load test pipeline. @jpayne3506 did the work recently, can we sync with him to add those changes to this one too ?

Reply:
Thanks for the info, I've updated this!